src/components/vice/admin/index.js

```diff
@@ -191,10 +210,133 @@ const VICEAdminSkeleton = () => {
+            </AppBar>
+
+            {orderOfTabs.map((tabName, index) => (
+                <TabPanel
+                    value={`${index}`}
+                    id={tabPanelID(tabName)}
+                    key={tabPanelID(tabName)}
+                >
+                    <CollapsibleTable
+                        rows={analysisRows}
+                        columns={columns[tabName]}
+                        title={msg(tabName)}
+                        showActions={tabName === "analyses"}
+                        handleExit={async (analysisID, externalID) => {
+                            const data = await mutantExit({ analysisID });
+                            setAnalysisRows(
+                                analysisRows.filter(
+                                    (obj) => obj.externalID !== externalID
+                                )
+                            );
+                            return data;
+                        }}
+                        handleSaveAndExit={async (analysisID, externalID) => {
+                            const data = await mutantSaveAndExit({
+                                analysisID,
+                            });
+                            setAnalysisRows(
+                                analysisRows.filter(
+                                    (obj) => obj.externalID !== externalID
+                                )
+                            );
+                            return data;
+                        }}
+                        handleExtendTimeLimit={async (
+                            analysisID,
+                            externalID
+                        ) => {
+                            const data = await mutantExtendTimeLimit({
+                                analysisID,
+                            });
+                            return data;
+                        }}
+                        handleUploadOutputs={async (analysisID, externalID) => {
+                            const data = await mutantUploadOutputs({
+                                analysisID,
+                            });
+                            return data;
+                        }}
+                        handleDownloadInputs={async (
+                            analysisID,
+                            externalID
+                        ) => {
+                            const data = await mutantDownloadInputs({
+                                analysisID,
+                            });
+                            return data;
+                        }}
+                    />
+                </TabPanel>
+            ))}
+        </TabContext>
+    );
+};
+
 const VICEAdmin = () => {
     const classes = useStyles();
 
-    const { status, data, error } = useQuery("vice-admin", getData);
+    const { status, data, error } = useQuery(getDataQueryName, getData, {
+        refetchInterval: 10000,
+    });
+
     const isLoading = status === "loading";
```

Comment:
I think `loading` is defined as a constant in global constants file. We can use that here. Also we can define `error` as a global constant while we are at it.

Reply:
I'll add it to the component-specific constants file. I'm wary of global or widely-scoped values.